scripts/node.sh

```diff
@@ -700,6 +710,31 @@ kill_node() {
    done
 }
 
+read_bls_pass() {
+   for f in ${blsfolder}/*.key
+   do
+      if [ ! -f $f ]; then
+         err 10 "could not find bls key file: $f"
+      fi
+      passfile=${blsfolder}/$(basename "${f%.*}").pass
+      if [ ! -f "$passfile" ]; then
+         unset -v passphrase
+         read -rsp "Enter passphrase for the BLS key file $f: " passphrase
+         echo ${passphrase} | tee $passfile
+         echo "Passphrase is temporarely saved to: $passfile, which will be deleted after the node is launched successfully!!!"
+      fi
+   done
+}
+
+rm_bls_pass() {
```

Comment:
we can give an option to users that they don't have to remove the passfile if they choose to. then we just give them a warning then.

Reply:
done.